src/components/Alerts/index.module.scss

```diff
@@ -1,42 +1,79 @@
 @import 'sizes.scss';
 @import 'colors.scss';
 
-.alert {
-    color: $brand-color-darkest;
-    &:not(.important) {
-        background: desaturate(lighten($brand-color, 30%), 60%);
-        .container {
-            background: lighten($brand-color, 35%);
+.alert .container {
+    max-width: $page-max-width;
+    margin: 0 auto;
+    padding: 0.75em 1em;
+
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    .message {
+        flex: 1;
+        text-align: center;
+
+        p, h1, h2, h3, h4, h5, h6 {
+            margin: 0;
+            font-size: 1em;
         }
     }
-    &:not(.important) + &:not(.important) .container {
-        margin-top: 2px;
+
+    .dismiss_button {
+        border: 1px solid transparent;
+        border-color: none;
+        background: none;
+        color: inherit;
+        padding: 0.4em;
+        height: 1.5em;
+        width: 1.5em;
+
+        display: flex;
+        align-items: center;
+
+        &::-moz-focus-inner {
+            border: none;
+            padding: 0;
+        }
+        .image {
+            width: 100%;
+            height: auto;
+        }
     }
+}
 
-    p {
-        margin: 0;
+.alert.level_info {
+    color: $brand-color-darkest;
+    background: desaturate(lighten($brand-color, 30%), 60%);
+    .container {
+        background: lighten($brand-color, 35%);
     }
 }
 
-.important {
+.alert.level_success {
+    color: #2f5e30;
+    background: desaturate($color: #dff0d8, $amount: 40%);
+    .container {
+        background: #dff0d8;
+    }
+}
+
+.alert.level_important {
+    color: $brand-color-darkest;
+    background: none;
```

Comment:
Why none? We could desaturate the red alert, so it is a mix between grey and red 😄

Reply:
Because it floats on the page, and I didn't think a coloured background looked quite right. The non-fixed (sticky) alerts have the appearance of a transparent overlay, but the affect is lost when it's not actually transparent.